mesonbuild/backend/ninjabackend.py

```diff
@@ -227,26 +229,25 @@ def strToCommandArg(c: T.Union[NinjaCommandArg, str]) -> NinjaCommandArg:
             self.depfile += '_UNQUOTED'
 
     @staticmethod
-    def _quoter(x, qf = quote_func):
-        if isinstance(x, NinjaCommandArg):
```

Comment:
Hmm, why did this think it could fail to be a NinjaCommandArg?

Reply:
https://github.com/mesonbuild/meson/commit/9cec5f3521407d50307eeef3c34cbd608949b0a1

Since the initial introduction of _quoter it has always only ever taken NinjaCommandArg, lol. Very good...

Reply:
I'm going to guess in the bad old days when we had no static type checking someone was just being defensive